rcl_yaml_param_parser/test/test_parse_yaml.cpp

```diff
@@ -43,6 +43,26 @@ TEST(test_file_parser, correct_syntax) {
   allocator.deallocate(path, allocator.state);
 }
 
+TEST(test_file_parser, string_array_with_quoted_number) {
+  rcutils_reset_error();
+  EXPECT_TRUE(rcutils_get_cwd(cur_dir, 1024));
+  char * test_path = rcutils_join_path(cur_dir, "test", allocator);
```

Comment:
nitpick: It appears all of the other tests are using local variables for the allocator. Maybe we can remove the global variable?